src/Files.App/Helpers/ItemListDisplayHelpers/BulkConcurrentObservableCollection.cs

```diff
@@ -159,26 +164,36 @@ public void ResetGroups(CancellationToken token = default)
             }
 
             // Prevents any unwanted errors caused by bindings updating
-            GroupedCollection.ForEach(x => x.Model.PausePropertyChangedNotifications());
-            GroupedCollection.Clear();
+            GroupedCollection?.ForEach(x => x.Model.PausePropertyChangedNotifications());
+            GroupedCollection?.Clear();
             AddItemsToGroup(collection, token);
         }
 
         private void AddItemsToGroup(IEnumerable<T> items, CancellationToken token = default)
         {
+            if (GroupedCollection is null)
+            {
+                return;
+            }
```

Comment:
```suggestion
            if (GroupedCollection is null)
                return;
```
We try to not use brackets for single line statements

Reply:
Ok, no problem. I'll change this even in your code styles, with your permission

![Screenshot (35)](https://user-images.githubusercontent.com/102259289/188894285-30025832-fbb5-44b7-87be-76d12a61f715.png)

Reply:
That would be great, thanks for noticing that!